src/components/TransactionsTracker/useTransactionsTracker.ts

```diff
@@ -30,4 +33,11 @@ export function useTransactionsTracker(props?: TransactionsTrackerType) {
     onMessage,
     address
   });
+
+  useEffect(() => {
+    timeoutRef.current = setTimeout(() => onMessage(), 30000);
```

Comment:
I wonder if this shouldn't be an interval, if transaction takes longer than 30 seconds, so we can do multiple calls